pkg/target/manager.go

```diff
@@ -63,6 +66,8 @@ type Manager interface {
 	UnsetTargetSeed() (string, error)
 	// UnsetTargetShoot unsets the garden shoot configuration
 	UnsetTargetShoot() (string, error)
+	// UnsetTargetControlPlane unsets the control plane flag
+	UnsetTargetControlPlane() (string, error)
```

Comment:
I don't understand why this method returns the targeted shoot. All other methods return the previous value which has been unset on the target object. I think this method should only return `errror` or `(bool, error)` but than bool value would always be true if no error occurs.

Reply:
changed interface